.github/workflows/publish.yml

```diff
@@ -225,7 +225,8 @@ jobs:
     needs: [ reactor-netty-core, tagRelease, tagMilestone, deploySnapshot ]
     if: always() # cleanup always run after all needed jobs, regardless of whether they were successful
     runs-on: ubuntu-20.04
-    permissions: write-all
+    permissions:
+        contents: write
```

Comment:
Is it `contents: write` or `actions: write`?

Reply:
it's `actions: write` ... mistake from me :slightly_frowning_face: 

thanks for catching, I will correct this.

Reply:
fixed in https://github.com/reactor/reactor-netty/pull/3158/commits/82249e4787cb3ea5ac440ba41defbe41b3b99f11